scripts/apps/authoring/authoring/preview-custom-field.tsx

```diff
@@ -13,16 +13,14 @@ interface IProps {
 export class PreviewCustomField extends React.PureComponent<IProps> {
     render() {
         const {item, field} = this.props;
-
-        const FieldType = fields.fields[field.custom_field_type];
+        const FieldType = fields[field.custom_field_type];
+        const value = get(item.extra, field._id);
```

Comment:
What does `lodash.get` do differently here than would `const value = item.extra[field._id]`?

Reply:
it won't fail if extra is not defined, that's about it